test/jasmine/tests/extend_test.js

```diff
@@ -1,3 +1,5 @@
+/*global Float32Array, Float64Array, Int16Array, Int32Array */
```

Comment:
Let's add these to the `test/jasmine/.eslintrc` file similar to the [`src/.eslintrc`](https://github.com/plotly/plotly.js/blob/master/src/.eslintrc#L6)